hushline/settings.py

```diff
@@ -177,9 +176,9 @@ def index() -> str | Response:  # noqa: PLR0911, PLR0912
 
             # Handle Change Password Form Submission
             if change_password_form.validate_on_submit():
-                if scrypt.verify(change_password_form.old_password.data, user.password_hash):
+                if user.check_password(change_password_form.old_password.data):
                     # Hash the new password using scrypt and update the user object
```

Comment:
Switch from scrypt to the preferred / recommended argon2.

Reply:
Addressed by (https://github.com/scidsg/hushline/pull/411/commits/6f53d35d4a1c4d29292f3bb7c093187192a02ea9)